README.md

```diff
@@ -76,6 +77,18 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
 2. Expand the **Extensions** section and click **OpenAPI**.
 3. Clear the checkbox **Alphabetically sort contents of OpenAPI explorer outlines**.
 
+
+### Preview OpenAPI documentation
+
+OpenAPI files can be easily converted to a human readable HTML documentation describing how to use your API. Our extension supports
+two popular libraries that perform such conversion: [SwaggerUI](https://swagger.io/tools/swagger-ui/) and [ReDoc](https://github.com/Redocly/redoc).
+
+To open the preview of API documentation execute relevant command from the Command Pallette (use Cmd-Shift-P or Ctrl-Shift-P to open):
```

Comment:
"To open the preview of the API documentation, execute the relevant command from the Command Palette (use Cmd+Shift+P or Ctrl+Shift+P to open the palette):"

Here, the context makes the nouns definite, so "the" needed for both, typo in the "palette". You have previously used pluses in the shortcuts (as Microsoft itself does, so stay consistent.